### DISCUSSION TITLE
[Enhancement] [message-box] ElMessageBox.prompt 希望input可以设置maxlength,showlimitword等字段

### DISCUSSION THREAD

**Discussion by zc-tired** [element-plus/element-plus#8792] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `message-box`

## Description

ElMessageBox.prompt 希望input可以设置maxlength,showlimitword等el-input可以设置的相关字段

### Additional comments

_(empty)_

**Comment by holly9715** [comment]
他应该加一个inputAttributes,通过这个对象设置input输入框属性

**Comment by snowbitx** [comment]
可以渲染`vnode`，你可以自己写个组件，也可以用h函数传入`ElInput`自定义props

<img width="1108" height="658" alt="image" src="https://github.com/user-attachments/assets/18d366f8-dcd6-43fa-953a-72afff57c93d" />
